tests/instruments/test_all_instruments.py

```diff
@@ -111,6 +111,7 @@ def test_adapter_arg(cls):
     "HP6634A",
     "HP8657B",
     "Keithley2000",
+    "Keithley2200",
```

Comment:
Don't add your instrument to the list of exceptions (instruments not yet changed), but make you instrument conform to the tests (see comment at init).
That means, revert this change.